service-map/src/main/java/fi/nls/oskari/map/layer/OskariLayerServiceIbatisImpl.java

```diff
@@ -240,6 +242,22 @@ public OskariLayer find(final String idStr) {
 
     }
 
+    public List<OskariLayer> findByIdList(final List<Integer> intList) {
+        final List<String> strList =  ConversionHelper.getStringListFromIntegers(intList);
```

Comment:
This is not needed as params are integers. You can also remove the method from ConversionHelper.